config/Migrations/20250125_OldTupiCodeRename

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Migrations\AbstractMigration;
+
+class OldTupiCodeRename extends AbstractMigration
+{
+    private $langColumns = [
```

Comment:
In theory you want to add the new column
```
'audios' => ['sentence_lang'],
```
let’s say, for the sake of completeness. But in practice it’s not really necessary because we don’t have any audio in Old Tupi.